operator/api/redpanda/v1alpha2/redpanda_clusterspec_types.go

```diff
@@ -1028,6 +1028,8 @@ type ClusterConfiguration vectorizedv1alpha1.ClusterConfiguration
 // SideCars configures the additional sidecar containers that run alongside the main Redpanda container in the Pod.
 type SideCars struct {
 	Image *RedpandaImage `json:"image,omitempty"`
+	// Insert some markers to hide the filed from CRD generation.
```

Comment:
@chrisseto I don't know how to hide field from CRD generation tool

Reply:
I detailed some options here: https://redpandadata.atlassian.net/browse/K8S-633?focusedCommentId=77776